src/vivarium_gates_nutrition_optimization/constants/results.py

```diff
@@ -88,8 +79,8 @@
     "AGE_GROUP": AGE_GROUPS,
     "CAUSE_OF_DEATH": CAUSES_OF_DEATH,
     "CAUSE_OF_DISABILITY": CAUSES_OF_DISABILITY,
-    "PREGNANCY_STATE": models.PREGNANCY_MODEL_STATES,
-    "PREGNANCY_TRANSITION": models.PREGNANCY_MODEL_TRANSITIONS,
+    "PREGNANCY_STATE": models.PREGNANCY_MODEL_STATES[1:],
```

Comment:
Why are you subsetting the list here?

Reply:
I thought this was causing an error in parsing column names, since `not_pregnant` is never reached by a tracked simulant.  But I just went back and checked, and it actually just fills those stratifications in by zero (which is the behavior I would expect). I will revert this